src/config/keys.ts

```diff
@@ -6,10 +6,19 @@ import { DEFAULT_THUMBNAIL_SIZE } from './constants';
 
 export const APPS_KEY = 'apps';
 export const ITEMS_KEY = 'items';
+export const SHORT_LINKS_KEY = 'shortLinks';
+export const SHORT_LINK_KEY = 'shortLink';
```

Comment:
I am not sure we need to plural here, as it is kind of hard to distinguish, but maybe this was to disambiguate with the `["shortLink", "items"]`(alias) and `["shortLinks", "items", "1234565z34723-2751843"]` ?

Reply:
It was to separate requests but in fact I can only keep one, because there are two keys that are built with the buildShortLink and buildShortLinksItemKey.